packages/calypso-e2e/src/lib/pages/media-page.ts

```diff
@@ -12,6 +13,8 @@ const selectors = {
 	items: '.media-library__list-item',
 	placeholder: '.is-placeholder',
 	editButton: 'button[data-e2e-button="edit"]',
+	addNewButton: 'input[type="file"]',
```

Comment:
A couple questions here:
1. Just to clarify - are we targeting the hidden input under the '.moxie-shim' div? I ask because this current selector seems a bit on the generic side, and am wondering if it's worth using something just a pinch more specific!
2. Because this isn't really the add new button, can we use a different property name? Something that suggests this is the selector for the hidden, test-only input field?

Reply:
1. I don't see a `.moxie-shim` div, but the gist is correct; the `upload` method will interact directly with the `input` element on the page. How about a selector like `input.media-library__upload-button-input`? 
2. Makes sense. Let's rename this to `fileInput` or the like?

Reply:
Those changes both sound great!